config/projects/omnibus-toolchain.rb

```diff
@@ -40,6 +40,9 @@
 override :bundler, version: "1.17.2"
 override :gtar, version: "1.32"
 
+# 1.1.1i+ builds on m1 mac
+override :openssl, version: "1.1.1i"
```

Comment:
We should probably get 1.1.1j  into omnibus-software and use that here

Reply:
Are you okay if that is a follow up? I mean, part of me wonders if 1.1.1j shouldn't be the default in omnibus-software.

Reply:
I updated to 1.1.1j. @tas50 explained to me that we keep the default to 1.0.2 because 1.1.1 doesn't support FIPS.